### PR TITLE
[Translation zh-tw] Shared > Udacity > ud882.html

### PR DESCRIPTION
**Pull request by limhenry**
The translation is based on Udacity (Chinese) website
https://cn.udacity.com/course/responsive-images--ud882

[Check out a review request at GitLocalize](https://gitlocalize.com/repo/107/zh-TW/review/395)

### CODE REVIEW

File: src/content/zh-tw/_shared/udacity/ud882.html [hunk 1]
```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.0 Transitional//EN" "http://www.w3.org/TR/REC-html40/loose.dtd">
```
Comment (petele) [inline src/content/zh-tw/_shared/udacity/ud882.html]: original does not contain the doctype, `html` or `body` tag. These files are included in to other files and should not include them.

**Comment by petele** [comment]
Also, can you sync with master? that'll fix the weird build issue.

**Comment by sotayamashita** [comment]
@limhenry Why did you delete forked repo?

**Comment by limhenry** [comment]
@sotayamashita  i deleted the forked repo? [this](https://github.com/limhenry/WebFundamentals/)?

**Comment by sotayamashita** [comment]
@limhenry I misunderstood. You have deleted the branch?

**Pull request closed without merging**